### PR TITLE
Completed Part 1 of the Tutorial Dapp

### PR DESCRIPTION
**Pull request by BboyAkers**
The content in this commit is in relation to the Initial Setup and Basic Actions.

### CODE REVIEW

File: docs/guide/create-dapp.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
+# Create A Simple Dapp
+::: tip 
+We will be building this [app](https://metamask.github.io/test-dapp/)
+:::
+
+## Project Setup
+Before you setup make sure you've visited and gone through our [Getting Started Guide](http://localhost:8080/guide/getting-started.html#getting-started)
+
+Make sure you have:
+1. Have the [MetaMask Extendsion](https://metamask.io/download.html) downlaoded.
```
Comment (jennypollack) [inline docs/guide/create-dapp.md:10]:
```suggestion
1. Have the [MetaMask Extension](https://metamask.io/download.html) downloaded.
```

File: docs/guide/create-dapp.md [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+npm run serve
+```
+
+Navigate to `http://localhost:9011`
+
+## Basic Action(Part 1)
+
+Now let's navigate into the contract.js file inside you start folder.
+
+Your file should look something like this. Don't worry about line 1-31.
```
Comment (jennypollack) [inline docs/guide/create-dapp.md:49]:
```suggestion
Your file should look something like this. Don't worry about lines 1-31.
```

**Comment by BboyAkers** [comment]
Done! Thanks for the catches

File: docs/guide/create-dapp.md [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+``` javascript
+const forwarderOrigin = 'http://localhost:9010'
+
+const initialize = () => {
+  //You will start here 
+}
+window.addEventListener('DOMContentLoaded', initialize)
+```
+
+As you can see here, as soon as the content in the DOM is loaded we are calling our initialized function. Now before we start writing any code we need to see what's on our task list for the first part of this app.
```
Comment (jennypollack) [inline docs/guide/create-dapp.md:60]:
```suggestion
As you can see here, as soon as the content in the DOM is loaded we are calling our initialize function. Now before we start writing any code we need to see what's on our task list for the first part of this app.
```

File: docs/guide/create-dapp.md [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+The first thing we need to do in our Dapp is to connect to our MetaMask Wallet.
+1. We need to create a function to see if the MetaMask Chrome extension is installed
+2. If MetaMask is not installed we:
+   1. Change our `connectButton` to `Click here to install MetaMask`
+   2. When clicking that button it should take us to a page that will allow us to install the extension
+   3. Disable the button
+3. If MetaMask is installed we:
+   1. Change our `connectButton` to `Connect`
+   2. When clicking that button it should allow us to connect to our MetaMask wallet
+   3. disable the button
```
Comment (jennypollack) [inline docs/guide/create-dapp.md:82]:
```suggestion
   3. Disable the button
```

File: docs/guide/create-dapp.md [hunk 5]
```diff
@@ -0,0 +1,261 @@
...
+Next we create a check function called `isMetaMaskInstalled` to see if the MetaMask extension is installed
+
+``` javascript
+const initialize = () => { 
+  //Basic Actions Section
+  const onboardButton = document.getElementById('connectButton')
+
+  //Created check function to see if the MetaMask extenstion is installed
+  const isMetaMaskInstalled = () => {
+    //Have to check the ehtereum binding on the window object to see if it's installed
```
Comment (jennypollack) [inline docs/guide/create-dapp.md:106]:
```suggestion
    //Have to check the ethereum binding on the window object to see if it's installed 
```

File: docs/guide/create-dapp.md [hunk 6]
```diff
@@ -0,0 +1,261 @@
...
+      //When the button is clicked we call this function to connect the users MetaMask Wallet
+      onboardButton.onclick = onClickConnect
+      //The button is now disabled
+      onboardButton.disabled = false
+    }
+  }
+  MetamaskClientCheck();
+```
+
+Now that we've created a function that will be called whenever we click the button to trigger a connection to our wallet and disabled the button. Let's create/dive into the `onClickConnect` function and build the logic inside of it.
```
Comment (jennypollack) [inline docs/guide/create-dapp.md:213]:
```suggestion
Now we've created a function that will be called whenever we click the button to trigger a connection to our wallet and disable the button. Next, let's dive into the `onClickConnect` function and build the logic inside of it.
```

File: docs/guide/create-dapp.md [hunk 7]
```diff
@@ -0,0 +1,261 @@
+# Create A Simple Dapp
+::: tip 
+We will be building this [app](https://metamask.github.io/test-dapp/)
+:::
+
+## Project Setup
+Before you setup make sure you've visited and gone through our [Getting Started Guide](http://localhost:8080/guide/getting-started.html#getting-started)
```
Comment (jennypollack) [inline docs/guide/create-dapp.md:7]:
```suggestion
Before you setup make sure you've visited and gone through our [Getting Started Guide](/guide/getting-started.html#getting-started)
```

Comment (jennypollack) [reply docs/guide/create-dapp.md:7]: ^is this for the dapp getting started guide, or the general getting started with metamask guide?

Comment (BboyAkers) [reply docs/guide/create-dapp.md:7]: This is the general getting started guide.

Comment (BboyAkers) [reply docs/guide/create-dapp.md]: I wanted to go back and make the general getting started more "hands on" than what is currently in the docs

**Pull request merged**